corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,7 +1,7 @@
 from corehq.util.elastic import es_index
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-DOMAIN_INDEX = es_index("hqdomains_20160531_1131")
+DOMAIN_INDEX = es_index("hqdomains_20160622_0645")
```

Comment:
here's one error:

```
FAIL: test_prod_config (testapps.test_elasticsearch.tests.test_prod_es_indices.ProdIndexManagementTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/testapps/test_elasticsearch/tests/test_prod_es_indices.py", line 27, in test_prod_config
    self.assertEqual(EXPECTED_PROD_INDICES, found_prod_indices)
AssertionError: Lists differ: [{'alias': 'case_search', 'typ... != [{'alias': u'case_search', 'ty...
First differing element 3:
{'alias': 'hqdomains', 'type': 'hqdomain', 'index': 'test_hqdomains_20160531_1131'}
{'alias': u'hqdomains', 'type': u'hqdomain', 'index': u'test_hqdomains_20160622_0645'}
Diff is 2710 characters long. Set self.maxDiff to None to see it.
```

I believe you also need to fix the settings test in `test_pillowtop` (which you can do by updating things in `all-pillow-meta.json`)